IGEL_USB_SERIAL_SUPPORT_HP_LDM350_DEVICES
	  Say Y here if you want to support the newer LDM350 devices also.

	  Source: IGEL
diff --git a/drivers/usb/serial/pl2303.c b/drivers/usb/serial/pl2303.c
index 22579d0d8ab8..8bfd656750c0 100644
--- a/drivers/usb/serial/pl2303.c
@@ -110,6 +110,9 @@ static const struct usb_device_id id_table[] = {
 	{ USB_DEVICE(HP_VENDOR_ID, HP_LM930_PRODUCT_ID) },
 	{ USB_DEVICE(HP_VENDOR_ID, HP_LM940_PRODUCT_ID) },
 	{ USB_DEVICE(HP_VENDOR_ID, HP_TD620_PRODUCT_ID) },
+#ifdef CONFIG_IGEL_USB_SERIAL_SUPPORT_HP_LDM350_DEVICES
+	{ USB_DEVICE(HP_VENDOR_ID, HP_LDM350_PRODUCT_ID) },
+#endif
 	{ USB_DEVICE(CRESSI_VENDOR_ID, CRESSI_EDY_PRODUCT_ID) },
 	{ USB_DEVICE(ZEAGLE_VENDOR_ID, ZEAGLE_N2ITION3_PRODUCT_ID) },
 	{ USB_DEVICE(SONY_VENDOR_ID, SONY_QN3USB_PRODUCT_ID) },
diff --git a/drivers/usb/serial/pl2303.h b/drivers/usb/serial/pl2303.h
index d60eda7f6eda..7029f2de179d 100644
--- a/drivers/usb/serial/pl2303.h
@@ -139,6 +139,9 @@
 #define HP_LCM220_PRODUCT_ID	0x3139
 #define HP_LCM960_PRODUCT_ID	0x3239
 #define HP_LD220_PRODUCT_ID	0x3524
+#ifdef CONFIG_IGEL_USB_SERIAL_SUPPORT_HP_LDM350_DEVICES
+#define HP_LDM350_PRODUCT_ID	0x3739
+#endif
 #define HP_LD220TA_PRODUCT_ID	0x4349
 #define HP_LD960TA_PRODUCT_ID	0x4439
 #define HP_LM940_PRODUCT_ID	0x5039
